### DISCUSSION TITLE
Error Unable to find component: [livewire-ui-modal]

### DISCUSSION THREAD

**Discussion by a100rougezarautz** [wire-elements/modal#290] [Q&A]
HI!!!

I followed the indications here and at https://philo.dev/laravel-modals-with-livewire/ on how to intall and run the component, but I get an error:

Livewire \ Exceptions \ ComponentNotFoundException           PHP 8.1.12           Livewire 9.22.1
Unable to find component: [livewire-ui-modal]

I added @livewire('livewire-ui-modal') to my layout file, and a call to alpinejs. 
![image](https://user-images.githubusercontent.com/118867549/203458226-0d2aa41b-c858-42d5-83be-550783fcce71.png)
![image](https://user-images.githubusercontent.com/118867549/203458276-9326522f-1304-4396-b6d3-ea7db462c4b7.png)

Any suggestions???

Thnx in advance

Igor

**Comment by kokoshneta** [comment]
Same issue here – did you ever manage to solve it?

**Reply by kokoshneta** [reply]
Oh, got it – the package service provider wasn’t being correctly discovered and injected, so I had to add it manually. In `/config/app.php`:

```php
return [
  // Other stuff…

  'providers' => [
    // Other providers…

    /*
     * Package service providers
     * /
    // Other package service providers…
    \LivewireUI\Modal\LivewireModalServiceProvider::class,
  ]
];
```
